### ISSUE TITLE
New object(s) : Lift

### ISSUE THREAD

**Issue by lambdaShade** [Electrical-Age/ElectricalAge#12]
3D models for versatile and convenient lifts are planned.
Priority : not too low... ;)

**Comment by bomdia** [comment]
not to low but very longterm any news on?

**Comment by TheAmazingSpy** [comment]
What sort of lift?

**Comment by Domi1993** [comment]
I think that it is a normal elevator there are normaly in house offices the one with sliding doors

**Comment by TheAmazingSpy** [comment]
NICE! Luckily, I can keep track on all the models (apple has an in-built previewer that can see the model files) I bet it would be realistic, and here are some suggestions: a winch to make it realistic, elevator music (should be able to find some royalty free) and a button panel

**Comment by Domi1993** [comment]
http://youtu.be/7F9Fie99Hu4

Here is a nice elevator music just ask the maker if he want to get this for The mod

**Comment by TheAmazingSpy** [comment]
Nah, maybe more like: https://www.youtube.com/watch?v=jj0ChLVTpaA

**Comment by Domi1993** [comment]
No thats bad

**Comment by Domi1993** [comment]
http://youtu.be/7F9Fie99Hu4

The maker of this elevator music has allowed to use it in the Mod

**Comment by SaphireLattice** [comment]
Why not just let people to select the mp3/ogg/wav file? Maybe a folder on
the server side from which players can choose an ogg/wav/etc file? Ofc that
means streaming..
And preferably a way to disable the music in the gui of elevator/etc. Not
every elevator has music irl.

**Comment by Domi1993** [comment]
Yes this will be nice, eventually make possible to insert own music on the players PC that he can put in a Folder and the mod will find it.

And the no music or music is a good idea too

**Comment by TheAmazingSpy** [comment]
Stop chatin' and get working! An elevator would be PERFECT!
